views/pages/urls/index.hbs

```diff
@@ -1,5 +1,19 @@
 {{!< center-container }}
 
+<nav aria-label="Page navigation example">
+    <ul class="pagination justify-content-end">
+        <li class="page-item {{#unless pagination.hasPrev}}disabled{{/unless}}">
+            <a class="page-link" href="?page={{ dec pagination.page }}" tabindex="-1">Previous</a>
+        </li>
+        {{#range pagination.pageCount}}
```

Comment:
do not add helpers unnecessarily too much. range is a bit pythonish way of doing things. we can simple achieve this using the foreach helpers that are inbuilt in hbs

Reply:
Actually pagecount is a number object here and foreach helper in hbs only iterates over iterables! That is why I needed to add this !

Reply:
Massage the data into an iterable ;)

Reply:
Done 🚀